### PR TITLE
make '-name' option of the 's_client' more generic

### PR DESCRIPTION
**Pull request by InfoHunter**
In response to issue #4422

And also make '-xmpphost' an alias of the '-name' option.

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

**Comment by InfoHunter** [comment]
Now travis-ci is okay. ping @openssl

File: doc/man1/s_client.pod [hunk 1]
```diff
@@ -514,6 +515,23 @@ specifies the host for the "to" attribute of the stream element.
 If this option is not specified, then the host specified with "-connect"
 will be used.
 
+This option is an alias of the B<-name> option since OpenSSL 1.1.1 for
```
Comment (levitte) [inline doc/man1/s_client.pod]: Please remove `since OpenSSL 1.1.1`.  If you must, add a remark about this in the `HISTORY` section

**Comment by levitte** [comment]
Other than that little detail, this looks fine to me

**Comment by InfoHunter** [comment]
new commit pushed to change what @levitte requested

**Comment by richsalz** [comment]
Merged, thanks!

**Pull request closed without merging**